### DISCUSSION TITLE
Cant used copilot after buying subscription

### DISCUSSION THREAD

**Discussion by bintangkurniawanpp** [community/community#118792] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

I've already buying subs but my balance on my debit card doesnt change, no matter how many times i tried, it still showed up that my subscription isnt upgraded

**Comment by davecheney** [comment, marked as answer]
@bintangkurniawanpp the card you have on file cannot be processed. 
I am sorry, we cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact

**Comment by quantumquantum12345** [comment]
i buy a package for 1 month subscription then why my account is not upgrade, i couldn't continue my chat with copilot in visual studio

**Comment by Uruglee2** [comment]
I also upgraded and never got the upgrade. Did you ever get a response from Github?

**Comment by abpenman25-WGC** [comment]
I upgraded my account, the payment went through, I can't use copilot?? Please fix.
